macros/unload.sql

```diff
@@ -56,6 +57,9 @@ where option is
   {% elif aws_key and aws_secret %}
   ACCESS_KEY_ID '{{ aws_key }}'
   SECRET_ACCESS_KEY '{{ aws_secret }}'
+  {% if aws_token %}
+    SESSION_TOKEN '{{ aws_token }}'
+  {% endif %}
```

Comment:
Just checked [the docs](https://docs.aws.amazon.com/redshift/latest/dg/copy-parameters-authorization.html):

> When SESSION_TOKEN is specified, you must also use ACCESS_KEY_ID and SECRET_ACCESS_KEY to provide temporary access key credentials

Makes sense!